app/pb_kits/playbook/pb_form/form_builder/form_field_builder.rb

```diff
@@ -12,6 +12,7 @@ def initialize(method_name, kit_name:)
             options[:skip_default_ids] = false unless options.key?(:skip_default_ids)
             options[:required] = true if props[:required]
             options[:placeholder] = props[:placeholder] || ""
+            options[:value] = props[:value]
```

Comment:
Are you using this?

Reply:
Had it setup such that value would be passed in directly as a prop.  I'll switch it to accept a value field in the options object instead.